lib/time_format.ex

```diff
@@ -0,0 +1,33 @@
+defmodule Logger.Backend.Humio.TimeFormat do
+  @moduledoc """
+  Helpers for formatting timestamps to ISO8601.
+  The Elixir logger provides the timestamp to backends as a {{year, month, day}, {hour, minute, second, millisecond}} tuple but without timestamp information.
+  """
+  use Timex
+
+  # if enabled, the logger's timestamp will be in UTC.  Otherwise it will be in local time
+  defp utc_log_enabled?() do
+    Application.get_env(:logger, :utc_log)
```

Comment:
shouldn't this config always exist? 
```suggestion
    Application.fetch_env!(:logger, :utc_log)
```

Otherwise I think it'd be better to set the default return value to false instead of nil:
```suggestion
    Application.get_env(:logger, :utc_log, false)
```